torch/quantization/fx/quantization_patterns.py

```diff
@@ -445,7 +445,29 @@ def convert(self,
                 convert_custom_config_dict: Dict[str, Any] = None) -> Node:
         if not self.all_node_args_are_tensors:
             return NotImplemented
-        return quantized_graph.node_copy(node, load_arg(quantized=torch.quint8))
+        if is_reference:
+            activation_post_process = \
+                self._maybe_get_last_node_only_observer(modules)
+            if activation_post_process is None:
```

Comment:
just curious, what's the context for this check? is this checking if the op is in floating point (so reference pattern is equivalent to the op)?  If yes, would it make sense to check the expected dtype instead?

Reply:
oh this is for cases like dynamic quantization and weight only quantization where no observer is required for the input and output activations. however dynamic quantization would quantize the activation internally so the numerics is still not matching for that case. right, for this case I think the dtype for activation should be torch.float, I can add the check

Reply:
will add this in a future PR